File: Module.Frontend.TPM/Client/model/tpm/PlanIncrementalReport/PlanIncrementalReport.js
Ext.define('App.model.tpm.planincrementalreport.PlanIncrementalReport', {
    extend: 'Ext.data.Model',
    idProperty: 'Id',
    breezeEntityType: 'PlanIncrementalReport',
    fields: [
        { name: 'Id', hidden: true },
        { name: 'ZREP', type: 'string', hidden: false, isDefault: true },
        { name: 'DemandCode', type: 'string', hidden: false, isDefault: true },
        { name: 'PromoNameId', type: 'string', hidden: false, isDefault: true },
        { name: 'LocApollo', type: 'string', hidden: false, isDefault: true },
        { name: 'TypeApollo', type: 'string', hidden: false, isDefault: true },
        { name: 'ModelApollo', type: 'string', hidden: false, isDefault: true },
        { name: 'WeekStartDate', type: 'date', hidden: false, isDefault: true, timeZone: 0, convert: dateConvertTimeZone }, // timeZone: 0, т.к. расширенный фильтр этого поля работает при таком значении
        { name: 'PlanProductIncrementalCaseQty', type: 'number', hidden: false, isDefault: true },
        { name: 'PlanUplift', type: 'number', hidden: false, isDefault: true },
		{ name: 'DispatchesStart', type: 'date', hidden: false, isDefault: true, timeZone: +3, convert: dateConvertTimeZone },
		{ name: 'DispatchesEnd', type: 'date', hidden: false, isDefault: true, timeZone: +3, convert: dateConvertTimeZone },
		{ name: 'Week', type: 'string', hidden: false, isDefault: true },
		{ name: 'Status', type: 'string', hidden: false, isDefault: true, defaultFilterConfig: getPlanIncrementalReportDefaultFilterStatus() },
		{ name: 'PlanProductBaselineCaseQty', type: 'number', hidden: false, isDefault: true },
		{ name: 'PlanProductIncrementalLSV', type: 'number', hidden: false, isDefault: true },
		{ name: 'PlanProductBaselineLSV', type: 'number', hidden: false, isDefault: true },
        { name: 'InOut', type: 'bool', hidden: false, isDefault: true },
        { name: 'IsGrowthAcceleration', type: 'bool', hidden: false, isDefault: true },
    ],
    proxy: {
        type: 'breeze',
        resourceName: 'PlanIncrementalReports',
        reader: {
            type: 'json',
            totalProperty: 'inlineCount',
            root: 'results'
        }
    },
});

function getPlanIncrementalReportDefaultFilterStatus() {
	var statuses = ['Approved', 'Planned', 'Started'];
	var value = Ext.create('App.extfilter.core.ValueList', statuses);

	var result = {
		value: value,
		operation: 'In'
	};
	return result;
}

File: Module.Persist.TPM/Model/DTO/PlanIncrementalReport.cs
using Core.Data;
using System;

namespace Module.Persist.TPM.Model.DTO {
    public class PlanIncrementalReport : IEntity<Guid>, ICloneable {
        public Guid Id { get; set; }
        public string ZREP { get; set; }
        public string DemandCode { get; set; }
        public string PromoNameId { get; set; }
        public string LocApollo { get; set; }
        public string TypeApollo { get; set; }
        public string ModelApollo { get; set; }
        public DateTimeOffset? WeekStartDate { get; set; }
        public double? PlanProductCaseQty { get; set; }
        public double? PlanUplift { get; set; }
        public DateTimeOffset? DispatchesStart { get; set; }
        public DateTimeOffset? DispatchesEnd { get; set; }
		public string Week { get; set; }
		public string Status { get; set; }
		public double? PlanProductBaselineCaseQty { get; set; }
		public double? PlanProductIncrementalLSV { get; set; }
		public double? PlanProductBaselineLSV { get; set; }
		public bool? InOut { get; set; }

		public object Clone()
		{
			return this.MemberwiseClone();
		}
	}
}
